jetty-core/jetty-deploy/src/test/java/org/eclipse/jetty/deploy/CoreContextHandlerTest.java

```diff
@@ -132,7 +132,7 @@ public void testExampleCoreDir(String dirname) throws Exception
         deploymentScanner.addWebappsDirectory(webapps);
         server.addBean(deploymentScanner);
         DeploymentScanner.EnvironmentConfig coreConfig = deploymentScanner.configureEnvironment("core");
-        coreConfig.setContextHandlerClass(CoreContextHandler.class.getName());
+        coreConfig.setClassLoaderFactoryClassName(CoreContextHandler.CoreContextClassLoaderFactory.class.getName());
```

Comment:
```suggestion
        coreConfig.setClassLoaderFactoryClass(CoreContextHandler.CoreContextClassLoaderFactory.class);
```

Same for the other occurrences.